src/components/with-information-model/index.tsx

```diff
@@ -10,6 +10,8 @@ import { DataFormat } from '../../types/enums';
 export interface Props {
   informationModel: InformationModel | null;
   informationModelRdfRepresentations: Partial<{ [key in DataFormat]: string }>;
+  isLoadingInformationModel: boolean;
```

Comment:
Blir denne brukt?

Reply:
Ja, for å ikke vise modell før den er lastet inn